hws/README.md

```diff
@@ -61,5 +53,5 @@ Required package
  - Python3
  - Scapy
  - Cython
- - Pyverbs (Install RDMA-CORE after Cython)
+ - Pyverbs (Install RDMA-CORE after Cython, planned to be removed in August release)
```

Comment:
August release is now, so should it be here?

Reply:
yes until it being released will update if any change happens.